strangerstudios/pmpro-avatax
# Avatax checkout implementation

### All Submissions:





### Changes proposed in this Pull Request:

Frontend implementation of the Avatax calculations based on [WooCommerce AvaTax](https://woocommerce.com/products/woocommerce-avatax/).
A totals section has been added in the PMPro checkout that is populated through an AJAX call. The new section has subtotal, tax, and total rows.
The tax is calculated on top of the Membership one-time price and is currently triggered only for US.


### How to test the changes in this Pull Request:
1. Make sure that you have the Avatax credentials setup, as well as membership levels, have attached an Avatax product code in the Tax Category > Product Category ID field.
2. Go to checkout
3. Fill a US-based address
4. Check that the totals section is updated with the right taxes based on the address and product code
5. Place order and check that order totals and taxes are sent correctly to Avalara

### Other information:

* The tax is also added to the recurring amounts. Have been tested with Stripe payment.



### Changelog entry

> PMPro checkout implementation of Avatax calculations.
